app/src/modules/insights/routes/dashboard.vue

```diff
@@ -387,7 +387,7 @@ const toggleZoomToFit = () => (zoomToFit.value = !zoomToFit.value);
 
 const refreshInterval = computed({
 	get() {
-		return unref(refreshIntervals)[props.primaryKey];
+		return unref(refreshIntervals)[props.primaryKey] ?? null;
```

Comment:
this is technically an unrelated fix, but it's due to `refresh-sidebar-detail` component no longer having `null` default value for it's modelValue prop, as changed in https://github.com/directus/directus/pull/18473/files#diff-46b86bd056cd2cf78779c0f14696747ca26beb888d0b8fb2d5534550e3bfada1.

This causes the Auto Refresh sidebar _within_ Insights dashboards to incorrectly show that it's configured when it's actually not:

https://user-images.githubusercontent.com/42867097/237010203-15b5f70e-4300-4f83-93e4-42834292c7f5.mp4

Reply:
Makes sense! 👍